packages/venia-ui/lib/components/MiniCart/ProductListing/productListing.js

```diff
@@ -0,0 +1,32 @@
+import React, { useMemo } from 'react';
+
+import LoadingIndicator from '../../LoadingIndicator';
+import Item from './item';
+import { mergeClasses } from '../../../classify';
+
+import defaultClasses from './productListing.css';
+
+const ProductListing = props => {
+    const { listings, loading, handleRemoveItem, classes: propClasses } = props;
```

Comment:
I think we should aim to use the same terminology as before, rather than introducing `listing` as a new term for the same thing.

This component should be `ProductList`, and `props.listings` should really just be `props.items`.